CHANGELOG.md

```diff
@@ -1,41 +1,42 @@
 ## Release (2024-01-09)
 
 ### Highlights
-
+- `core`: [v0.7.5](core/CHANGELOG.md#v0.72024-01-09)
```

Comment:
Shouldn't this be
```suggestion
- `core`: [v0.7.5](core/CHANGELOG.md#v075-2024-01-09)
```
?

Reply:
Yes it should, not sure what happened here...